storage/streams/src/main/java/io/apicurio/registry/streams/events/KafkaEventSink.java

```diff
@@ -0,0 +1,103 @@
+package io.apicurio.registry.streams.events;
+
+import java.time.Instant;
+import java.util.Optional;
+import java.util.Properties;
+import java.util.UUID;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.header.Headers;
+import org.apache.kafka.common.header.internals.RecordHeaders;
+import org.apache.kafka.common.serialization.Serdes;
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.apicurio.registry.events.EventSink;
+import io.apicurio.registry.utils.RegistryProperties;
+import io.apicurio.registry.utils.kafka.AsyncProducer;
+import io.apicurio.registry.utils.kafka.ProducerActions;
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.eventbus.Message;
+
+/**
+ * @author Fabian Martinez
+ */
+@ApplicationScoped
+public class KafkaEventSink implements EventSink {
+
+    private static final Logger log = LoggerFactory.getLogger(KafkaEventSink.class);
+
+    @Inject
+    @RegistryProperties(
+            value = {"registry.streams.common", "registry.streams.events"},
+            empties = {"ssl.endpoint.identification.algorithm="}
+    )
+    Properties producerPproperties;
```

Comment:
Is producerPproperties or producerProperties?

Reply:
that's a typo